csharp/src/Ice/Internal/IdleTimeoutTransceiverDecorator.cs

```diff
@@ -60,7 +62,6 @@ public int initialize(Buffer readBuffer, Buffer writeBuffer, ref bool hasMoreDat
 
         if (op == SocketOperation.None) // connected
         {
-            rescheduleReadTimer();
```

Comment:
Redundant with the new "enableIdleCheck()" called when switching to StateActive.